### DISCUSSION TITLE
The account was cancelled due to not changing the password for a long time.

### DISCUSSION THREAD

**Discussion by jxnu-liguobin** [apache/pekko#1411] [Q&A]
I once received an email asking me to click to continue using my password, but I don't know why I couldn't click. I have received email now. 

`You've been removed from the "The Apache Software Foundation" team "pekko committers"`

how to solve it? 🤪 
@mdedetrich @He-Pin @raboof

**Comment by He-Pin** [comment]
I have no idea ,but I think @pjfanning can help.

**Reply by pjfanning** [reply]
It might be a temporary measure. An incident was reported to the ASF Security and Infrastructure teams yesterday. It is still being investigated. Is it possible that some third party has managed to get your credentials?

**Reply by jxnu-liguobin** [reply]
I don't know either. I received an email from Apache reminding me that my password is about to expire, but I can't click it. How can I retrieve my account now?

![image](https://github.com/user-attachments/assets/aa67d6cc-9d33-4ad7-9769-b26130c45222)

**Reply by pjfanning** [reply]
That is not an Apache email as far as I know. We have our own Apache email software called Apache Ponymail. We don't use 3rd party software.

**Reply by jxnu-liguobin** [reply]
> That is not an Apache email as far as I know. We have our own Apache email software called Apache Ponymail. We don't use 3rd party software.

Maybe it is. Do you know what to do next? 

![image](https://github.com/user-attachments/assets/ad6d9f1b-9f51-4fab-9183-946fcc0d5c6b)

**Reply by pjfanning** [reply]
@raboof should we get this 'roundcube' email forwarded to someone so that they can check it's links? It looks like a phishing attack that can be used on other Apache committees.

**Reply by raboof** [reply]
It's definitely phishing. I'm not sure how to protect against those (there are many variations). Perhaps INFRA could make an announcement that they never ask for email/domain validation... but I'm not sure that would have a lasting impact.

**Reply by jxnu-liguobin** [reply]
I have added double checked for my original email and updated my password.

**Reply by jxnu-liguobin** [reply]
How can I recover my account?🤒 @pjfanning @raboof

**Reply by pjfanning** [reply]
The breach was pretty serious. The people who stole your password were using it maliciously. I will follow up with Infrastructure team but it could take a few days or more. You can still submit PRs and review other PRs using GitHub.

**Reply by jxnu-liguobin** [reply]
ping @pjfanning

**Reply by pjfanning** [reply]
I haven't heard back from the Infrastructure team yet. I'll try to follow up with them.

**Reply by jxnu-liguobin** [reply]
> I haven't heard back from the Infrastructure team yet. I'll try to follow up with them.

Is it still a phishing email
？  😂
![image](https://github.com/user-attachments/assets/7ee04574-7b5c-4a2f-adc5-88fe569a00ac)

**Reply by pjfanning** [reply]
Apache Software Foundation do not send emails to get you to change your password. And membership info is public. So easy for phishing attackers. Members need to carefully check any links provided in the emails.

**Reply by jxnu-liguobin** [reply]
Excuse me,@pjfanning, is there any progress?

**Reply by pjfanning** [reply]
This is out of my hands. You were asked to prove your identity to the ASF Infrastructure team and they are not convinced that you have proved this. Contact them again.

**Reply by pjfanning** [reply]
https://issues.apache.org/jira/browse/INFRA-26106

**Reply by jxnu-liguobin** [reply]
> https://issues.apache.org/jira/browse/INFRA-26106

I am unable to log in 🥲

**Reply by pjfanning** [reply]
You were provided with the email address to contact - the person in Infra who is dealing with this. I have no say or control over this.
